pkg/controller/proxyconfig/validation.go

```diff
@@ -14,7 +13,10 @@ import (
 
 	configv1 "github.com/openshift/api/config/v1"
 	"github.com/openshift/cluster-network-operator/pkg/names"
+	"github.com/openshift/cluster-network-operator/pkg/platform"
 	"github.com/openshift/cluster-network-operator/pkg/util/validation"
+	"golang.org/x/net/proxy"
```

Comment:
Q: would you consider calling this import `netProxy` ? I say that because it is a bit 'scary' when looking at functions that remain using `proxy` as parameters, like: `func validateReadinessEndpoint(caBundle []*x509.Certificate, proxy, endpoint string) error {`